Contribute/how-to-write-links.md

```diff
@@ -70,12 +72,12 @@ You do not have to create anchors. They're automatically generated at publishing
   `[link](#the-text-of-the-H2-section-separated-by-hyphens)`
   `[Create cache](#create-cache)`
 
-- To link to an anchor in another article in the same subdirectory:
+- To link to an anchor in another article in the same directory:
 
   `[link text](article-name.md#anchor-name)`
   `[Configure your profile](media-services-create-account.md#configure-your-profile)`
 
-- To link to an anchor in another service subdirectory:
+- To link to an anchor in another directory:
```

Comment:
This is really no different, in both cases the only difference is adding the fragment.  Suggest we drop this bullet and drop "in the same directory" from the preceding bullet.